jobs/build/merge_ocp/Jenkinsfile

```diff
@@ -19,7 +19,7 @@ node {
                         name: 'VERSIONS',
                         description: 'CSV list of versions to run merge on.',
                         $class: 'hudson.model.StringParameterDefinition',
-                        defaultValue: "3.9,3.10,3.11,4.0"
+                        defaultValue: "3.9,3.10,3.11,4.0,4.1"
```

Comment:
These are the current supported building versions right? This feels like it should be in a `commonlib` type place.

Reply:
Fair. I'll see what I can do.

Reply:
Do we actually need to merge 4.1, as long as we're building 4.1 from master? I would rather merge master->master and not merge 4.1 until master becomes 4.2

Reply:
That's what setting `CURRENT_MASTER = "4.1"` does... 4.1 will be master -> master

Reply:
yeah, nevermind me :)

Reply:
they're not the current supported building versions; they're the versions that still merge from origin. i don't think we will need to know that anywhere else any longer.